test_cases/feedback_pass.json

```diff
@@ -36,11 +36,11 @@
       "expected": {
         "properties": [
           {
-            "text": "31 세종대로 서울시청 신관동, Seoul",
+            "text": "31 세종대로 서울시청 본관동, Seoul",
```

Comment:
Agreed, it's because I branched off https://github.com/pelias/acceptance-tests/pull/50 by accident